main.py

```diff
@@ -58,11 +60,11 @@ def reg_trainer():
     x_test = torch.linspace(-2., 2, config.num_test_points).reshape(-1, 1)
     for m_name, model in models.items():
         model.net.load_state_dict(torch.load(model.save_model_path, map_location=torch.device(DEVICE)))
-        y_test = model.evaluate(x_test, config.num_test_points)
```

Comment:
for my understanding 'config.num_test_points' became the attribute 'test_samples'?

Reply:
not quite: num_test_points is the number of x values that form the test set (see line 60), while test_samples is the number of predictions made for a given test point. I think we previously incorrectly had num_test_points here instead of test_samples (not that it should make much difference).